source/common/upstream/upstream_impl.h

```diff
@@ -850,6 +854,8 @@ class ClusterInfoImpl : public ClusterInfo,
   absl::optional<envoy::config::cluster::v3::Cluster::RingHashLbConfig> lb_ring_hash_config_;
   absl::optional<envoy::config::cluster::v3::Cluster::MaglevLbConfig> lb_maglev_config_;
   absl::optional<envoy::config::cluster::v3::Cluster::OriginalDstLbConfig> lb_original_dst_config_;
+  absl::optional<envoy::config::cluster::v3::Cluster::DeterministicApertureLbConfig>
+      lb_daperture_config_;
```

Comment:
Maybe just `lb_aperture_config_`? "daperture" reads like a typo everytime I see it.